.github/pull_request_template.md

```diff
@@ -1,28 +1,47 @@
-## Brief Information
+<!-- 
+🚨NOTE🚨 Please complete the following tasks before submitting a pull request:
 
-This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):
+- [ ] Read the CONTRIBUTING doc. see: https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md
+- [ ] Sign the CLA. see: https://cla-assistant.io/hypertrons/hypertrons-crx
+-->
 
-- [ ] build
-- [ ] ci
-- [ ] docs
-- [ ] feat
-- [ ] fix
-- [ ] perf
-- [ ] refactor
-- [ ] test
+### Brief
 
-Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):
+This pull request is in the type of:
+<!-- 
+- `build`: Changes related to the build system, such as dependencies, packaging, and compiling.
+- `ci`: Changes related to continuous integration, such as configuration files and scripts.
+- `docs`: Changes related to documentation, such as README files and API documentation.
+- `feat`: New features or enhancements to existing features.
+- `fix`: Bug fixes or patches to existing features.
+- `perf`: Changes related to performance improvements.
+- `refactor`: Changes that improe code quality or structure without changing functionality.
+- `test`: Changes related to testing, such as adding or modifying unit tests. 
 
-- keyword #xxx
+Ref: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type
 
-## Details
-<!-- What did you do in this PR?  -->
+Example:
+
+    This pull request is in the type of: build
+-->
+
+Related issues:
+<!-- 
+Link the issue to the PR.
+
+Ref: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
 
-## Checklist
+Example:
 
-- [ ] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
-- [ ] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)
+    Related issues:
+
+    - fix #xx1
+    - address #xx2 (mention but not close)
+-->
```

Comment:
How about this way:

```text
Related issues:

- close #111 (mentnion and link)
- close #222
- #333 (just mention)
```

Reply:
Sure, I see the `address` is used in other communities, it's not a keyword, but means to handle something. But shorter for better, it's also nice.